test/Dataworker.loadData.ts

```diff
@@ -381,6 +460,63 @@ describe("Dataworker: Load data used in all functions", async function () {
       expect(data1.bundleFillsV3[repaymentChainId][l1Token_1.address].fills.length).to.equal(1);
       expect(data1.bundleDepositsV3).to.deep.equal({});
     });
+    it("Validates fill from lite chain against old deposit", async function () {
+      // For this test, we need to actually send a deposit on the spoke pool
+      // because queryHistoricalDepositForFill eth_call's the contract.
+
+      // Send a deposit.
+      const depositObject = await depositV3(
+        spokePool_1,
+        destinationChainId,
+        depositor,
+        erc20_1.address,
+        amountToDeposit,
+        erc20_2.address,
+        amountToDeposit
+      );
+      const depositBlock = await spokePool_1.provider.getBlockNumber();
+      // Construct a spoke pool client with a small search range that would not include the deposit.
+      spokePoolClient_1.firstBlockToSearch = depositBlock + 1;
+      spokePoolClient_1.eventSearchConfig.fromBlock = spokePoolClient_1.firstBlockToSearch;
+      await spokePoolClient_1.update();
+      const deposits = spokePoolClient_1.getDeposits();
+      expect(deposits.length).to.equal(0);
+
+      // Mock the config store client to include the lite chain index.
+      mockConfigStore.updateGlobalConfig(
+        GLOBAL_CONFIG_STORE_KEYS.LITE_CHAIN_ID_INDICES,
+        JSON.stringify([spokePoolClient_1.chainId])
+      );
+      await mockConfigStore.update();
+      // Ensure that our test has the right setup.
+      expect(mockConfigStore.liteChainIndicesUpdates.length).to.equal(1);
+      mockConfigStore.liteChainIndicesUpdates[0].timestamp = depositObject.quoteTimestamp - 1;
+      expect(mockConfigStore.liteChainIndicesUpdates[0].timestamp).to.be.lt(depositObject.quoteTimestamp);
+      expect(repaymentChainId).to.not.eq(spokePoolClient_1.chainId);
```

Comment:
```suggestion
      expect(repaymentChainId).to.not.eq(originChainId);
```

same suggestion as above

Reply:
https://github.com/across-protocol/relayer/pull/1629/commits/a489074b3e00abe73d27929ddb2c69b953ad86b4